makefile.example

```diff
@@ -106,3 +106,8 @@ fix:
 # Check migration files without running them
 validate:
 	bash -c "goose -dir=$(MIGRATION_DIR) validate"
+
+# ==========================================================
+# Seeds database with data
+seed: 
+	bash -c "bash seeds/seed_accounts.sh"
```

Comment:
can use `make seed` command to seed database

Reply:
missing elements